app/routes/file-download.js

```diff
@@ -0,0 +1,36 @@
+const { blobServiceClient } = require('../storage')
+const config = require('../config/storage')
```

Comment:
Suggestion to match FCP style

```suggestion
const { storageConfig }  = require('../config')
```